membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -56,7 +60,9 @@ class TouchpointComponents(stage: String)(implicit system: ActorSystem) {
 
   lazy val ukStripeService = new StripeService(tpConfig.stripeUKMembership, RequestRunners.loggingRunner(metrics("stripe")))
   lazy val auStripeService = new StripeService(tpConfig.stripeAUMembership, RequestRunners.loggingRunner(metrics("stripe")))
-  lazy val stripeServicesByPaymentGateway: Map[PaymentGateway, StripeService] = Seq(ukStripeService, auStripeService).map(s => s.paymentGateway -> s).toMap
+  lazy val allStripeServices = Seq(ukStripeService, auStripeService)
```

Comment:
really, it's the stripe config that changes, not the whole stripe service, but I guess that's a bigger symptom of the touchpoint backend bloat